### ISSUE TITLE
Repeating section expression properties not working

### ISSUE THREAD

**Issue by michalkubat** [formly-js/ngx-formly#745]
```
[ ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[x] support request
```

**Current behavior**
Expression properties defined in repeating section are not reacting on changes. For instance, I have a checkbox that sets other input in repeating section as disabled. Anyway, whenever I uncheck that checkbox, nothing happens (unchecking the checkbox should set the other input as disabled).

**Minimal reproduction of the problem with instructions**
I used repeating section component provided in example. Here is used config.

```
[{
        "fieldGroupClassName": "display-flex",
        "fieldGroup": [{
                "className": "flex-1",
                "type": "input",
                "key": "name",
                "templateOptions": {
                    "label": "Name",
                    "required": true,
                    "maxLength": 255
                }
            }
        ]
    },
    {
        "key": "section",
        "type": "repeat-section",
        "fieldArray": {
            "fieldGroupClassName": "display-flex",
            "fieldGroup": [{
                    "className": "",
                    "type": "checkbox",
                    "key": "cityDisabled",
                    "templateOptions": {
                        "label": ""
                    }
                },
                {
                    "className": "flex-1",
                    "type": "input",
                    "key": "team",
                    "templateOptions": {
                        "label": "Team",
                        "disabled": false,
                        "required": true
                    }
                },
                {
                    "className": "flex-1",
                    "type": "input",
                    "key": "city",
                    "templateOptions": {
                        "label": "City",
                        "required": true,
                        "disabled": false
                    },
                    "expressionProperties": {
                        "templateOptions.disabled": "model.cityDisabled"
                    }
                }
            ]
        }
    }
]
```

* **Angular version:** 5.2.5

* **Browser:** Chrome 
 
* **Language:** TypeScript 2.4.2

* **Node (for AoT issues):** `node --version` =   v9.5.0

**Comment by chaosLegacy** [comment]
Hi sir, i'm using Bootstrap Horizontal according in your doc i've added my horizontal-wrapper.ts and all works fine, my problem is that I have a select and a textarea in my form that i can't add multiple type at theme for example 
`{
      key: 'recurrente',
      type: 'horizontalInput', // 'select works here but i'm gonna lose my horizontalInput'
      templateOptions: {
        type:'select', // this is doesn't work here 
        label: 'Toutes la semaine',
        options: [
          {label: 'Tous les jours ouvrables', value: '1'},
          {label: 'Quotidienne', value: '2'},
          {label: 'Hebdomadaire', value: '3'},
          {label: 'Mensuelle', value: '4'},
          {label: 'Annuelle', value: '5'},
        ],
      },
    },` as you can see the type for select, textarea,... don't work inside templateOptions, is there any way i can fix this ? 
thank you very helpful tool

**Comment by aitboudad** [comment]
@chaosLegacy try with
```ts
{ 
  key: 'recurrente', 
  type: 'select',
  wrappers: ['fieldset', 'horizontalWrapper'],
  ...
}
```

or define a new type for select `horizontalSelect` like you did for `horizontalInput`.
```ts
FormlyModule.forRoot({
  types: [
    ...
    { name: 'horizontalSelect', extends: 'select', wrappers: ['fieldset', 'horizontalWrapper'] }
  ],
}),
```

**Comment by chaosLegacy** [comment]
thanks bro its works like charm thanks again for your great help

**Comment by aitboudad** [comment]
fixed in `2.0.0-rc.32`.

Please make sure to update your `RepeatTypeComponent`: https://formly-js.github.io/ngx-formly/examples/advanced/repeating-section

```ts
import { Component, OnInit } from '@angular/core';
import { FieldArrayType, FormlyFormBuilder } from '@ngx-formly/core';

@Component({
  selector: 'formly-repeat-section',
  template: `
    <div *ngFor="let field of field.fieldGroup; let i = index;">
      <formly-group
        [model]="model[i]"
        [field]="field"
        [options]="options"
        [form]="formControl">
      </formly-group>
      <div class="col-md-2">
        <button class="btn btn-danger" type="button" (click)="remove(i)">Remove</button>
      </div>
    </div>
    <div style="margin:30px 0;">
      <button class="btn btn-primary" type="button" (click)="add()">Add More Investments</button>
    </div>
  `,
})
export class RepeatTypeComponent extends FieldArrayType {
  constructor(builder: FormlyFormBuilder) {
    super(builder);
  }
}
```
